hazelcast-client/src/main/java/com/hazelcast/client/impl/HazelcastClientInstanceImpl.java

```diff
@@ -233,7 +236,12 @@ public HazelcastClientInstanceImpl(ClientConfig config,
         clusterService = new ClientClusterServiceImpl(this, addressProviders);
 
         int maxAllowedConcurrentInvocations = properties.getInteger(MAX_CONCURRENT_INVOCATIONS);
-        callIdSequence = new CallIdSequence.CallIdSequenceFailFast(maxAllowedConcurrentInvocations);
+        long backofftimeoutMs = properties.getLong(INVOCATION_BACKOFF_TIMEOUT_MILLIS);
+        if (maxAllowedConcurrentInvocations == Integer.MAX_VALUE) {
```

Comment:
the client does not have any option to explicitly disable backpressure. max in-flight operation is set the MAX_VALUE by default thus it's effectively disabled.